core/cap-0035.md

```diff
@@ -151,15 +151,15 @@ index 8d746391..a272f744 100644
  
  struct TrustLineEntry
  {
-@@ -337,6 +357,27 @@ case CLAIMABLE_BALANCE_ID_TYPE_V0:
+@@ -337,6 +356,27 @@ case CLAIMABLE_BALANCE_ID_TYPE_V0:
      Hash v0;
  };
  
 +enum ClaimableBalanceFlags
 +{
 +    // If set, the issuer account of the asset held by the claimable balance may
 +    // clawback the claimable balance
-+    CLAWBACK_ENABLED_FLAG = 0x1
++    CB_CLAWBACK_ENABLED_FLAG = 0x1

```

Comment:
In other parts of the XDR we use full word prefixes, can we do same here, and on the trustline?
```suggestion
+    CLAIMABLE_BALANCE_CLAWBACK_ENABLED_FLAG = 0x1

```
And then on the trustline rename `CLAWBACK_ENABLED_FLAG` to `TRUSTLINE_CLAWBACK_ENABLED_FLAG`?

Reply:
Done